src/main/java/seedu/duke/Finux.java

```diff
@@ -0,0 +1,85 @@
+package seedu.duke;
+
+import seedu.duke.command.CreditScoreMap;
+import seedu.duke.command.Command;
+import seedu.duke.command.CommandHandler;
+import seedu.duke.command.ExitCommand;
+import seedu.duke.exception.FileLoadingException;
+import seedu.duke.parser.ParserHandler;
+import seedu.duke.record.RecordList;
+import seedu.duke.storage.Storage;
+import seedu.duke.ui.Ui;
+
+import java.util.ArrayList;
+
+import static seedu.duke.common.Constant.FINUX_LOGGER;
+
+public class Finux {
+    private Ui ui;
+    private RecordList records;
+    private Storage storage;
+    private CommandHandler commandHandler;
+    private ParserHandler parserHandler;
+    private CreditScoreMap creditScoreMap;
+
+    /**
+     * Main entry-point for Finux application.
+     */
+    public static void main(String[] args) {
+        new Finux().run();
+    }
+
+    /**
+     * Runner for the FINUX Application.
+     */
+    private void run() {
+        start();
+        commandLooper();
+        end();
+    }
+
+    /**
+     * Starts the main application.
+     */
+    private void start() {
+        try {
+            ui = new Ui();
+            storage = new Storage();
+            parserHandler = new ParserHandler();
+            commandHandler = new CommandHandler();
+            storage.loadFile();
```

Comment:
![image](https://user-images.githubusercontent.com/61925875/113076978-c6eddf80-9202-11eb-96f4-89af303ed38b.png)


Was the call to loadFile() of the Finux class intentionally omitted?
Should "ArrayList" be "ArrayList//<Record//>" for the returning arrow after the call to getRecordListData() of Storage?

Reply:
Thanks for pointing this out! We have changed it.